src/components/comment/comment.scss

```diff
@@ -25,6 +26,7 @@
 /// those comments visually with their parent.
 .c-comment.is-replying::after,
 .c-comment--thread::after {
+  background-color: var(--theme-color-border-kbd);
```

Comment:
Can we create a border property specifically for this? `kbd` refers to the `kbd` element, so I'm not sure it's the most intuitive thing to use here. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd

If it happens to be the right color, maybe we just need to rename it wherever it's referenced to something more general.

Reply:
> I'm not sure it's the most intuitive thing to use here

Haha probably not 😆. I added a property specifically for the border color.